Backend/app/history/serializers.py

```
@@ -0,0 +1,315 @@
+            journal_entry.save()
+        
+        return journal_entry
+
+
+class ActivityLogSerializer(serializers.ModelSerializer):
+    """Serializer for activity logs"""
+    mood_impact = serializers.ReadOnlyField()
+    energy_impact = serializers.ReadOnlyField()
+    
+    @extend_schema_field(serializers.CharField)
+    def mood_impact(self, obj):
```

Ruff (0.12.2)

Redefinition of unused `mood_impact` from line 114

(F811)